modules/aws-outposts-load-balancer-clb.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-load-balancer-clb_{context}"]
+= Using AWS Classic Load Balancers in an AWS VPC cluster extended into an Outpost
+
+Outposts racks cannot run AWS Classic Load Balancers, but Classic Load Balancers in the AWS VPC cluster can attach to edge compute resources in the Outpost.
+As a result, Classic Load Balancers on the VPC cluster might schedule pods on either of these node types.
+
+If you want to prevent the Classic Load Balancer in the VPC cluster from attaching to Outpost edge compute nodes, you can apply labels to the edge compute nodes and configure the Classic Load Balancer to avoid scheduling on nodes with the applied labels.
+
+[NOTE]
+====
+If you do not need to prevent the Classic Load Balancer in the VPC cluster from attaching to Outpost edge compute nodes, you do not need to complete these steps.
+====
+
+.Prerequisites
+
+* You have extended an AWS VPC cluster into an Outpost.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have installed the AWS Load Balancer Operator and created the AWS Load Balancer Controller.
+
+.Procedure
+
+. List the nodes on the Outpost by running the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                   STATUS    ROLES     AGE       VERSION
+node1.example.com      Ready     worker    7h        v1.28.5
+node2.example.com      Ready     worker    7h        v1.28.5
+----
+//Does this need some additional filtering to ensure we are only listing Outposts nodes and not the VPC nodes too?
+
+. For each, label the node with a key-value pair by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node_name> <key_name>=<value>
+----
++
+#is there a way to label them all at once in case there are a bunch? Something related to the `--all` parameter here? https://docs.openshift.com/container-platform/4.14/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-updating_nodes-nodes-working#
```

Comment:
If we want to add several labels at one time, we can use `oc label node  <node_name>  <key_name_1>=<value_1> ... <key_name_n>=<value_n>`
If we want to label several nodes at one time, may we can use `oc label node  <key_name_1>=<value_1> -l=<node_label>`, then can add new label for the nodes with same label,`--all` will label all nodes including master nodes and outpost nodes.

Reply:
I think we would want to only label the Outposts workers, so they can't use `--all` in this case. Is that right? 

For this step and for `oc get nodes` above, I wonder if there is a way to filter to list only Outposts nodes (and similarly, to apply labels to all Outposts nodes but no others). But I don't know much about working with nodes, so maybe this is not possible (or not simple).